data_fusion/plugins/modules/procflows/data_fusion.py

```diff
@@ -529,6 +529,16 @@ def call(fnames, command_line_args=None):
         command_line_args, {"METADATA": fuse_data["final"]["metadata_xobj"]}
     )
 
+    if command_line_args.get("output_checker_kwargs") is not None:
+        output_checker_kwargs = command_line_args["output_checker_kwargs"]
+    else:
+        output_checker_kwargs = {
+            "image": {"image_threshold": "medium"},
+            "geotiff": {},
+            "netcdf": {},
+            "text": {},
+        }
```

Comment:
output_checker_kwargs = command_line_args.get("output_checker_kwargs", {})
Remove explicit plugin names.

Reply:
It looks like this still needs to be resolved, correct? @evrose54 @mindyls

Reply:
This has been updated now @jsolbrig @mindyls